### PR TITLE
fix(dict): Don't correct "accreting"

### PR DESCRIPTION
**Pull request by tklauser**
"accreting" is the present participle of "accrete", see https://en.wiktionary.org/wiki/accreting. Don't correct it to "accrediting".

### CODE REVIEW

**Comment by epage** [comment]
This was added in #315 with no comment about where it came from

File: crates/typos-dict/assets/words.csv [hunk 1]
```diff
@@ -687,7 +687,6 @@ accredidation,accreditation
 accreditied,accredited
 accreditted,accredited
 accress,access
-accreting,accrediting
```
Comment (epage) [inline crates/typos-dict/assets/words.csv:690]: I worry about this being added again in the future.  Could you add this to https://github.com/crate-ci/typos/blob/master/crates/typos-dict/assets/allowed.csv so we prevent this from happening again?

Comment (tklauser) [reply crates/typos-dict/assets/words.csv:690]: Thanks for pointing this out. Now added to `allowed.csv`.

File: crates/typos-dict/assets/allowed.csv [hunk 2]
```diff
@@ -5,3 +5,4 @@ referer,http header field
 deques,noun
 dequeues,verb
 ons,so `add-ons` works
+accreting,verb
```
Comment (epage) [inline crates/typos-dict/assets/allowed.csv:8]: Could you provide more detail, like "verb of accrete"?

While I was similarly brief with `deques` and `dequeues`, that was more in context of each other (before we corrected one to the other)

**Pull request merged**